docs/source/java/flight_sql_jdbc_driver.rst

```diff
@@ -114,6 +114,21 @@ parameters are:
      - null
      - When TLS is enabled, the password for the certificate store
 
+   * - tlsRootCerts
+     - null
+     - Path to PEM-encoded root certificates for TLS - use this as
+       an alternative to ``trustStore``
+
+   * - clientCertificate
+     - null
+     - Path to PEM-encoded client mTLS certificate when the Flight
+       SQL server requires client verification.
+
+   * - clientKey
+     - null
+     - Path to PEM-encoded client mTLS key when the Flight
```

Comment:
Maybe clientPrivateKey would be more clear?

Reply:
agreed - I'll make the change...

Reply:
Oh - I realized I used `clientKey` because the Java Flight client has it named that way:
https://github.com/apache/arrow/blob/57f643c2cecca729109daae18c7a64f3a37e76e4/java/flight/flight-core/src/main/java/org/apache/arrow/flight/FlightClient.java#L661C1-L661C1

Are you ok with the name (for consistency's sake)?